### DISCUSSION TITLE
Currently Having Issues With all Project Pages Redirecting to User Page.

### DISCUSSION THREAD

**Discussion by portlandhodl** [community/community#138585] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am attempting to publish a Github Pages project page for my api documentation. This has been met with what I think is an issue.

[Target Page](https://github.com/russeree/slipstream-api)
[Target Repo w/ Page](https://russeree.github.io/slipstream-api)

Even though in settings I have actived the github page against the master with ```/``` The page still is just a redirect to [My user page](https://russeree.github.io)

**Comment by LiteBrite82** [comment, marked as answer]
Looks like its working now.

**Reply by portlandhodl** [reply]
So for anyone that experiences this, all you have to do to get the site to render properly on Chome / Firefox is break the cache using CTRL+F5
